spyder/plugins/updatemanager/workers.py

```diff
@@ -118,28 +188,58 @@ def __init__(self, stable_only):
         super().__init__()
         self.stable_only = stable_only
         self.latest_release = None
-        self.releases = None
         self.update_available = False
         self.error = None
+        self.channel = None
 
-    def _check_update_available(self):
+    def _check_update_available(
+            self,
+            releases: list[Version],
+            github: bool = True
```

Comment:
```suggestion
        self,
        releases: list[Version],
        github: bool = True
```

Remove extra indentation.